### PR TITLE
Fixed top menu layout

### PR DESCRIPTION
**Pull request by ishcherbakov**
| Questions | Answers |
| --- | --- |
| Branch? | develop |
| Description? | Fix top menu layout: <br/>\- Set fixed size and position according to design<br/>\- Improve responsive behaviour |
| Type? | bug fix |
| Category? | FO |
| BC breaks? | no |
| Deprecations? | no |
| How to test? | Add multiple items (categories / subcategories / CSM pages / etc) to the top menu. Test behaviour with different screen window size |

#### Important guidelines
- Make sure [your local branch is up to date](https://help.github.com/articles/syncing-a-fork/) before commiting your changes!
- Your code MUST respect [our Coding Standards](http://doc.prestashop.com/display/PS16/Coding+Standards) (for code written in PHP, JavaScript, HTML/CSS/Smarty/Twig, SQL)!
- Your commit name MUST respect our [naming convention](http://doc.prestashop.com/display/PS16/How+to+write+a+commit+message)!

### CODE REVIEW

File: themes/classic/templates/_partials/header.tpl [hunk 1]
```diff
@@ -21,8 +21,11 @@
             <img class="logo img-responsive" src="{$shop.logo}" alt="{$shop.name}">
           </a>
         </div>
-        <div class="col-md-10">
-          {hook h='displayTop'}
+        <div class="col-md-10 position-static">
+          <div class="row">
+            {hook h='displayTop'}
+            <div class="clearfix"></div>
```
Comment (julienbourdeau) [inline themes/classic/templates/_partials/header.tpl]: Shouldn't it be something like this ?

```smarty
<div class="clearfix">
  {hook h='displayTop'}
</div>
```

**Comment by julienbourdeau** [comment]
Thanks!
You don't have to commit built assets (theme.js and theme.css) if you want to avoid rebasing your PR, someone will commit them at some point.

**Comment by Quetzacoalt91** [comment]
ping @mickaelandrieu for the last comment. :wink:

File: themes/classic/templates/_partials/header.tpl (hunk 1 above)
Comment (ishcherbakov) [inline themes/classic/templates/_partials/header.tpl]: For me it leads to the same result, but keeps DOM structure more clear.

**Comment by vincentbz** [comment]
image
image
image

Instead of (currently)
image
image
image

seems better :)
status : 'QA approved' thanks !

**Comment by vincentbz** [comment]
Tested and ok, thanks !

**Comment by vincentbz** [comment]
test

**Comment by vincentbz** [comment]
Tested and ok, thanks !

**Comment by mickaelandrieu** [comment]
Hi,

**Comment by mickaelandrieu** [comment]
Hi,

**Comment by vincentbz** [comment]
Tested and ok, thanks !

**Comment by mickaelandrieu** [comment]
Thank you Ivan !

File: themes/classic/templates/_partials/header.tpl (hunk 1 above)
Comment (julienbourdeau) [inline themes/classic/templates/_partials/header.tpl]: I think the result might be slightly different, but not visible.
http://stackoverflow.com/questions/8554043/what-is-clearfix

Anyway it's not very important :D

**Pull request merged**